autoblocks/_impl/util.py

```diff
@@ -38,7 +38,7 @@ def encode_uri_component(s: str) -> str:
 async def gather_with_max_concurrency(
     max_concurrency: int,
     coroutines: List[Coroutine],
-) -> None:
+):
```

Comment:
why drop this? can you update the type signature now that you're returning

Reply:
Waiting till we merge change to drop python 3.8